docs/OnnxReleases.md

```diff
@@ -118,3 +118,6 @@ The ONNX project, going forward, will plan to release roughly on a four month ca
 **Remove old onnx-weekly packages on TestPyPI**
 * Once ONNX has been released on PyPI, remove all previous versions of [onnx-weekly package](https://test.pypi.org/project/onnx-weekly/#history) on TestPyPI to save space.
 * Steps: Login and go [here](https://test.pypi.org/manage/project/onnx-weekly/releases/) -> Choose target package -> Options -> Delete.
+
+**Bump opset version for ai.onnx**
+* Bump opset version for ai.onnx domain in `onnx/defs/operator_sets.h` and `onnx/defs/schema.h` in advance. For example, this [demo PR](https://github.com/onnx/onnx/pull/4134/files).
```

Comment:
Minor nit: I was slightly confused by `in advance`. May be useful to use different wording to clarify that this happens after the release, not before. (I guess this subsection is after-release, but still.) How about changing it to
```
for use by future operator additions and changes
```